packages/bits/spec/virtual-camera/CameraSettings.ts

```diff
@@ -19,10 +19,10 @@ export class CameraSettings {
     private async crop(): Promise<void> { this.currentSettings.fullframe = false }
```

Comment:
I just noticed that the filename doesn't follow the kebab-case naming convention